doc/_templates/sections/announcement.html

```diff
@@ -0,0 +1,11 @@
+{% set header_classes = ["bd-header-announcement", "container-fluid"] %}
+{%- if '+' in release %}
```

Comment:
Do we have access to all variables from conf.py here? if so would it be better to do `is_release_build` here?

https://github.com/matplotlib/matplotlib/blob/1cc50f659a3a488c6ef9a265448a48a99956f411/doc/conf.py#L34C1-L34C17